crates/jstz_cli/src/sandbox/mod.rs

```diff
@@ -1,52 +1,44 @@
 use anyhow::{Ok, Result};
 use clap::Subcommand;
-use log::info;
-use nix::{
-    sys::signal::{kill, Signal},
-    unistd::Pid,
-};
 
 mod daemon;
 
 mod consts;
 
 pub use consts::*;
 
-use crate::{config::Config, error::bail_user_error};
+use crate::config::Config;
 
 #[derive(Debug, Subcommand)]
 pub enum Command {
     /// 🎬 Starts the sandbox.
-    Start,
+    Start {
+        /// Detach the process to run in the background.
+        #[clap(long, short, default_value = "false")]
+        detach: bool,
+        /// Run the sandbox in the background without showing any output.
+        #[clap(long, short, default_value = "false", hide = true)]
+        background: bool,
```

Comment:
Two flags? What is the difference here

Reply:
It appears --background is an internal flag for starting the sandbox as another process. It would be good to document that it is internal

Reply:
It is hidden, thus it won't be shown in help or documentation as there is no reason for the user to be able to access it.

Reply:
I'm aware, what I meant was that we should document that this flag _wont_ be used by the user (the documentation is for us, not the user)